src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -2414,8 +2477,8 @@ def lagrange_polynomial(self, points, algorithm="divided_difference", previous_r
 #                 P += (F[i] * prod)
 #             return P
 
-        # using Neville's method for recursively generating the
-        # Lagrange interpolation polynomial
+# using Neville's method for recursively generating the
+# Lagrange interpolation polynomial
```

Comment:
Please revert. The indentation is useful and such a change can just create (merge) conflicts.